pset-functions.js

```diff
@@ -174,6 +256,31 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 4:')
 
 // Add your code below this line
+function isPalindrome(word){
+  // split the word into characters
+  let wordChar=word.split("")
+  let reverseWord=[]
+  //console.log(wordChar)
+  // add the characters into reverseWord in reverseorder
+  for (let i = 0; i < wordChar.length; i++) {
+    reverseWord[wordChar.length-i-1]=wordChar[i]
+  }
+  // Output the word and the revers for debugging
+  console.log("Original word: " + word)
+  console.log("Revers word: " + reverseWord.join(""))
+  // check if we have a Palindrome
+  if (word===reverseWord.join("")) {
+    return true
+  }
+}
+
+const test="racecar"
+if (isPalindrome(test))
+  {console.log("is a Palindrome")
+  } else
+  {console.log("maybe undefined")
+  }
+
```

Comment:
@AtleLohrmann Good work on this problem. Good job using that` isPalindrome`  function to compare the word as is and reversed. I see your looping through each character in reverse and adding it to your array `reverseWord`.  where you split the word into the array `wordChar`, there is a method for arrays called `reverse()` which would reverse the array for you. It's a good tool and you could check out some information on it [here](https://www.w3schools.com/jsref/jsref_reverse.asp)One suggestion for lines 277 to 281 you could wrap that up in a function and call it. take a look at this solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L242)